scripts/keri/cf/verifier-config-docker.json

```diff
@@ -16,7 +16,8 @@
   ],
   "trustedLeis": [],
   "allowedEcrRoles": [
-    "EBA Data Submitter"
+    "EBA Data Submitter",
```

Comment:
Some time soon the vlei-verifier repo configs won't contain these EBA specific values, perhaps we should refactor to a verifier-config-eba.json or verifier-config-docker-eba.json. Lets decide when that would be appropriate.